src/apiCalls.js

```diff
@@ -10,35 +10,40 @@ const listTransformationsEndpoint = `${serverEndpoint}/transformations`;
 const listLibrariesEndpoint = `${serverEndpoint}/libraries`;
 
 const defaultHeader = {
-  "user-agent": "transformationAction"
-}
+  "user-agent": "transformationAction",
+};
 
 async function getAllTransformations() {
+  core.info(`Getting all transformations from upstream`);
+
   return axios.default.get(listTransformationsEndpoint, {
```

Comment:
minor: create a axios client (auth + headers) upfront and use it everywhere

Reply:
I can make this change.